crates/core/app/src/app/mod.rs

```diff
@@ -181,34 +184,35 @@ impl App {
             num_candidate_txs
         );
 
-        let mut proposal_size_bytes = 0u64;
+        // This is a node controlled parameter that is different from the homonymous
+        // mempool's `max_tx_bytes`. Comet will send us raw proposals that exceed that
+        // limit, presuming that a subset of those transactions will be shed.
```

Comment:
```suggestion
        // This is a node controlled parameter that is different from the homonymous
        // mempool's `max_tx_bytes`. Comet will send us raw proposals that exceed this
        // limit, presuming that a subset of those transactions will be shed.
```

'that limit' sounded like it was referring to mempool's homonymous parameter again.